web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-highlighted-co.html.twig

```diff
@@ -43,7 +43,8 @@
     'os2loop-highlighted-container',
     'paragraph--type--' ~ paragraph.bundle|clean_class,
     view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
-    not paragraph.isPublished() ? 'paragraph--unpublished'
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    drupal_field('os2loop_shared_subject', 'node', paragraph.parent_id.value)['0']['#options'].entity.os2loop_taxonomy_color_picker.value
```

Comment:
We should add a check for if this value (and indexes etc.) actually is defined; probably just appending `|default('')` will work.

Reply:
Added |default('')